drafter.gyp

```diff
@@ -281,7 +281,7 @@
         "src/ConversionContext.h",
         "src/ElementInfoUtils.h",
         "src/ElementComparator.h",
-
+	"src/ContentTypeMatcher.h",
```

Comment:
Perhaps GitHub is showing this wrong, but looks like mixing tabs here where rest of file is spaces.